modules/composer-ffmpeg/src/main/java/org/opencastproject/composer/impl/ComposerServiceImpl.java

```diff
@@ -148,9 +164,29 @@
   /** The formatter for load values */
   private static final DecimalFormat df = new DecimalFormat("#.#");
 
+  /** Configuration for process-smil transition duration - TODO: use smil transition element */
```

Comment:
Maybe just create a JIRA ticket for the "SMIL transition element" feature and remove the TODO from the comment. At the end of the day, source code is not necessarily the optimal place to put feature requests ;-)